openstudiocore/src/model/ElectricLoadCenterDistribution.cpp

```diff
@@ -33,8 +33,8 @@
 #include "ElectricalStorage_Impl.hpp"
```

Comment:
I'm assuming the changes in this file have nothing to do with your op scheme troubles.